run/image_processing/main.tf

```diff
@@ -88,24 +88,18 @@ output "blurred_bucket_name" {
 # [END cloudrun_service_image_processing_buckets]
 
 # [START cloudrun_service_image_processing_crservice]
-resource "google_cloud_run_service" "default" {
+resource "google_cloud_run_v2_service" "default" {
   name     = "pubsub-tutorial"
   location = "us-central1"
   template {
-    spec {
-      containers {
-        image = "gcr.io/cloudrun/hello"
-        env {
-          name  = "BLURRED_BUCKET_NAME"
-          value = google_storage_bucket.imageproc_output.name
-        }
+    containers {
+      image = "gcr.io/cloudrun/hello"
```

Comment:
The tutorial doesn't use this region tag, but it probably should (https://cloud.google.com/run/docs/tutorials/image-processing#shipping_the_code). Do we think we should add a placeholder like "YOUR_CONTAINER_IMAGE" instead of "gcr.io/cloudrun/hello" for tutorial purposes?

Reply:
added